cla_public/templates/checker/result/face-to-face.html

```diff
@@ -74,4 +74,9 @@ <h1>{{ _('A legal adviser may be able to help you') }}</h1>
     {% include '_ga-outcome-pageview.html' %}
   {% endblock %}
 
+  {% block ga_pageview -%}
+    ga('set', 'title', 'Results for [postcode redacted] | Seek legal advice');
+    ga('set', 'location', window.location.href.replace(/postcode=[^&]+/gi, 'postcode=redacted'));
```

Comment:
Would it be possible/meaningful to unit test this javascript snippet?

Reply:
Yep certainly. Though I can't run the e2e tests repo sans ECR access 😔 I could build the image individually but I'm running way out of scope on LGA-47. Will clarify next steps 👍

Reply:
Agreed with Emma to ticket at top of backlog. Already written so should land soon.